cd/application-deployment/dev/vaec-celery-task-definition.json

```diff
@@ -228,6 +228,10 @@
         {
           "name": "COMP_AND_PEN_DYNAMODB_NAME",
           "value": "dev-bip-payment-notification-table"
+        },
+        {
+          "name": "VA_PROFILE_V3_COMBINE_CONTACT_INFO_AND_PERMISSIONS_LOOKUP",
```

Comment:
are we not adding this to the rest of the environments?

Reply:
We are now!